mail_room.gemspec

```diff
@@ -20,6 +20,11 @@ Gem::Specification.new do |gem|
   gem.add_dependency             "net-imap", ">= 0.2.1"
   gem.add_dependency             "oauth2", "~> 1.4.4"
 
+  # Pinning io-wait to 0.1.0, which is the last version to support Ruby < 3
+  gem.add_dependency              "io-wait", "~> 0.1.0"
```

Comment:
We might be able to drop this now that `io-wait` v0.2.1 is out: https://github.com/ruby/net-imap/issues/24#issuecomment-973678392

Reply:
@stanhu Thanks for pointing this out. I dropped the dependency in the latest commit.